app/main/db/queries.py

```diff
@@ -560,3 +560,54 @@ def get_all_transferring_bodies():
     for body in Body.query.all():
         bodies.append(body.Name)
     return bodies
+
+
```

Comment:
can you add tests for these functions and/or adjust/add 1 test for each browse view at the route level to show that the breadcrumbs are in the html for the browse, browse_body, browse_series, browse_consignment, record

Reply:
test cases are already written for each browse view to verify breadcrumb values